### PR TITLE
Filter and note editor added

### PR DESCRIPTION
**Pull request by ashishkatiyar2404**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -2,3 +2,15 @@
   height: 90vh;
   background-color: beige;
```
Comment (ShrutiLakhani) [inline src/App.css:3]: Please use color variables

**Comment by Prashantrajput11** [comment]
clean code and UI. Good to merge

File: src/Components/Filters/FilterHook.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  const { filterInitialState } = useFilter();
+  const { notes } = useNote();
+  const newFilterNotes = [...notes];
+
+  const { sortByDate, sortByTags, sortByPriority } = filterInitialState;
+
+  const getFilterByTags = (notesArr, tags) => {
+    console.log("get Filter by tags");
+    if (tags.length > 0) {
+      console.log("get Filter by tags if k andr");
```
Comment (ShrutiLakhani) [inline src/Components/Filters/FilterHook.js:14]: Please remove unnecessary console.logs

File: src/Components/Navbar/Navbar.css [hunk 3]
```diff
@@ -3,22 +3,21 @@
...
-  background-color: burlywood;
-} */
-
 .navbar__credential {
-  background-color: beige;
+  background-color: purple;
   margin-right: 2rem;
-  color: rgb(24, 23, 23);
+  /* color: rgb(24, 23, 23); */
+  color: white;
```
Comment (ShrutiLakhani) [inline src/Components/Navbar/Navbar.css:18]: Use Color variables

**Review by Prashantrajput11 — APPROVED** [review]
Clean code looks good to merge

**Review by ShrutiLakhani — APPROVED** [review]
Code looks good. You may want to remove unnecessary consoles and its good to merge.

File: src/Components/Filters/FilterHook.js (hunk 2 above)
Comment (ashishkatiyar2404) [reply src/Components/Filters/FilterHook.js:14]: ok

File: src/Components/Navbar/Navbar.css (hunk 3 above)
Comment (ashishkatiyar2404) [reply src/Components/Navbar/Navbar.css:18]: sure

**Pull request merged**